app/assets/stylesheets/mo/_form_elements.scss

```diff
@@ -116,5 +116,12 @@ a.filter-only {
 .name-radio {
   label {
     font-weight: normal;
-  } 
+  }
+}
+
+.btn-glyphicon-only {
```

Comment:
@nimmolo 
- Is there a better name for this style?
- Can its substance be improved?
- Does it belong in this file or in app/assets/stylesheets/mo/_top_nav.scss?

Reply:
@JoeCohen - I changed some things to make it resemble the current links. 

The class will be also useful in Bootstrap 4-5, it's `btn-link`. 
It's for when the element must be a button, but you want it to resemble a link. 
I agree that the top nav logout might be better as a button, and i'm looking to rethink all that stuff soon (before upgrading to BS4) but it might make that simpler to keep it as is for now.

Reply:
(I think `.btn-glyphicon-only` is not necessary with these changes)